src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -437,6 +438,12 @@ private void CompileNamedType(NamedTypeSymbol containingType)
                                 hasStaticConstructor = true;
                             }
 
+                            // Replaced methods are not included in GetMembers().
+                            foreach (var replaced in method.GetReplacedMethods())
+                            {
+                                CompileMethod(replaced, extendedOrdinal, ref processedInitializers, synthesizedSubmissionFields, compilationState);
```

Comment:
Added a comment to `extendedOrdinal` that it is used to ensure replaced methods have unique ids.